responsive-test-utils/src/main/java/dev/responsive/kafka/api/ResponsiveTopologyTestDriver.java

```diff
@@ -97,22 +97,21 @@ public ResponsiveTopologyTestDriver(
         topology,
         config,
         initialWallClockTime,
-        new TTDCassandraClient(
-            new TTDMockAdmin(baseProps(config), topology),
-            mockTime(initialWallClockTime))
+        new TTDCassandraClient(new TTDMockAdmin(baseProps(config), topology))
     );
   }
 
   /**
-   * Advances the internal mock time used for Responsive-specific features such as ttl, as well
-   * as the mock time used for various Kafka Streams functionality such as wall-clock punctuators.
+   * Advances the internal mock time which can be used to trigger some Kafka Streams
+   * functionality such as wall-clock punctuators, or force Responsive to flush
+   * its internal buffers.
```

Comment:
Thanks to [this ](https://github.com/responsivedev/responsive-pub/pull/376/files#r1820067634)"hack" we no longer require users to use this method to advance wall clock time and manually flush the CommitBuffer in their TTD apps. Generally this is only used for triggering punctuation in Streams so most people wouldn't already be using this in their apps, and will advance time via record timestamps